lib/_stream_readable.js

```diff
@@ -69,28 +72,20 @@ function prependListener(emitter, event, fn) {
     emitter._events[event] = [fn, emitter._events[event]];
 }
 
-function ReadableState(options, stream, isDuplex) {
-  options = options || {};
-
-  // Duplex streams are both readable and writable, but share
-  // the same options object.
-  // However, some cases require setting options to different
-  // values for the readable and the writable sides of the duplex stream.
-  // These options can be provided separately as readableXXX and writableXXX.
-  if (typeof isDuplex !== 'boolean')
-    isDuplex = stream instanceof Stream.Duplex;
```

Comment:
This was here in case there are people calling this constructor directly (since it is made available on the `Readable` constructor as a static property). If we remove this, we should definitely do a CITGM to be on the safe side.